lib/providers/quoters/RfqQuoter.ts

```diff
@@ -56,19 +62,33 @@ export class RfqQuoter implements Quoter {
           log.error({ validationError: validated.error }, 'RfqQuoterErr: POST quote response invalid');
           metrics.putMetric(`RfqQuoterValidationErr`, 1);
         } else {
+          const getPortionResponse = await this.portionProvider.getPortion(
```

Comment:
I think you can move the `getPortion` call to the promise array above. Then within each promise resolve from `Promise.allSettled(requests)`, you don't need to repeatedly call `getPortion`.

Out of all 3 remote calls, `getPortion` is likely the fastest response